SerhiiBuza/HW6/task2.py

```diff
@@ -0,0 +1,18 @@
+# use loop while
+user_name = input("Enter your USER NAME")
```

Comment:
according to the condition of the problem, the solution should be like this

user_name = input('Hello, please input your Log in:')

while user_name != 'First':
    user_name = input('Error: wrong username, please try one more time. Username:')
else:
    print('Greeting. Access granted!!!', user_name)